internal/tofu/context_plan2_test.go

```diff
@@ -4533,6 +4533,171 @@ import {
 	}
 }
 
+func TestContext2Plan_importToInvalidDynamicAddress(t *testing.T) {
```

Comment:
I didn't change any of these tests, just reordered and moved it higher in the file